docker/Makefile

```diff
@@ -1,9 +1,13 @@
 .PHONY: alpine amazonlinux ubuntu fedora ubuntu-current-jedi final
 
-GERBIL_VERSION := master
+$(info the branch is $(GERBIL_VERSION))
```

Comment:
Why the info twice?

Reply:
To determine if the variable was read in from ENV, or from block.